### PR TITLE
Change: subscription template

### PR DESCRIPTION
**Pull request by ned29**

### CODE REVIEW

File: src/main/resources/templates/subscriptionsTemplate.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 [{
-    "subscriptionName" : "Subscription1",
+    "subscriptionName" : "Subscription1__Only_numbers_letters_and_underscore_are_allowed",
```
Comment (tobiasake) [inline src/main/resources/templates/subscriptionsTemplate.json]: what is this used for? for test? If test, then it should be placed under src/test/resources.

Comment (ned29) [reply src/main/resources/templates/subscriptionsTemplate.json]: Regarding my story, Sergio requested include to template which digital are allowed, please look into my story

Comment (ned29) [reply src/main/resources/templates/subscriptionsTemplate.json]: Should I do it or it will do someone else?
Also i think i should close this pr.

Comment (evasiba-ericsson) [reply src/main/resources/templates/subscriptionsTemplate.json]: This should be as a comment in the subscription on WIKI

**Pull request closed without merging**